src/Stan/Inspection/AntiPattern.hs

```diff
@@ -368,3 +371,20 @@ stan0214 = mkAntiPatternInspection (Id "STAN-0214") "use 'compare'" UseCompare
         [ "Rewrite code to use single 'compare' instead of many comparison operators"
         ]
     & severityL .~ Performance
+
+-- | 'Inspection' — Slashes in paths @STAN-0215@.
+stan0215 :: Inspection
+stan0215 = mkAntiPatternInspection (Id "STAN-0215") "Slashes in paths" (FindAst pat)
+    & descriptionL .~ "Usage of '/' or '\' in paths results in the errors on different operation systems"
+    & solutionL .~
+        [ "{Extra dependency} Use '</>' operator from 'filepath'"
+        ]
+    & severityL .~ Error
+  where
+    pat :: PatternAst
+    pat =   opApp pathLit filepathOperator (?)
+        ||| opApp (?) filepathOperator pathLit
+
+    pathLit :: PatternAst
+    pathLit = PatternAstConstant (ContainStr "/")
+        ||| PatternAstConstant (ContainStr "\\")
```

Comment:
I think it should be 4 slashes... Just to have a more robust check.

Reply:
Good catch! 👍